app/forms/webauthn_setup_form.rb

```diff
@@ -52,6 +52,22 @@ def consume_parameters(params)
     @client_data_json = params[:client_data_json]
     @name = params[:name]
     @platform_authenticator = (params[:platform_authenticator].to_s == 'true')
+    @transports = valid_transports_or_nil!(params[:transports]&.split(','))
+  end
+
+  def valid_transports_or_nil!(transports)
+    known_transports, unknown_transports = transports.partition do |transport|
+      WebauthnConfiguration::VALID_TRANSPORTS.include?(transport)
+    end
+
+    if unknown_transports.present?
+      NewRelic::Agent.notice_error(
+        RuntimeError.new('Unknown WebAuthn transports'),
+        custom_params: { unknown_transports: },
+      )
```

Comment:
let's write to `events.log` or `production.log`, errors in newrelic always cause more panic than intended

Reply:
maybe we just add an `unknown_transports` to the extra analytics params so they get logged with the form?

Reply:
Ah, yeah, I'll look to something like that. I do think there should be some urgency to this, since it is very unexpected and we'd want to add the unexpected transport rather immediately. I was looking at this in a same way we [consider unknown docauth alerts](https://github.com/18F/identity-idp/blob/a38217828c26802c63e7ad643d4a04e8708a6978/app/services/doc_auth/error_generator.rb#L179-L183), which I seem to recall had previously gone to NewRelic, but apparently go to CloudWatch now instead.

Reply:
I think a worry I'd have is making sure we have some active monitoring for that. Maybe it needs to be on some dashboard somewhere.

Reply:
Logging via analytics in 50d0ce0.